File: pyjl/cases/byte_literals.py
#!/usr/bin/env python3


if __name__ == "__main__":
    # byte literals
    assert b"foo" != b"bar"
    assert b'"' == b'"'
    assert b"'" == b"'"
    assert b"\xBBfoo" == b"\xBBfoo"
    print("OK")

File: tests/expected/julia/byte_literals.jl
if abspath(PROGRAM_FILE) == @__FILE__
    @assert(b"foo" != b"bar")
    @assert(b"\"" == b"\"")
    @assert(b"'" == b"'")
    @assert(b"\xbbfoo" == b"\xbbfoo")
    println("OK")
end
